src/reynir_correct/errtokenizer.py

```diff
@@ -462,14 +469,21 @@ def error_detail(self) -> Optional[str]:
         """Return the detailed description of this error, if any"""
         return getattr(self._err, "detail", None)
 
+    @property
+    def error_references(self) -> Optional[List[str]]:
+        """Return references to Icelandic Standards, if any"""
+        return getattr(self._err, "references", None)
+
     def add_corrected_meanings(self, m: Sequence[BIN_Tuple]) -> None:
         """Add alternative BÍN meanings for this token, based on a
         suggested spelling correction"""
         assert self.kind == TOK.WORD
         # We assume that the token has already been marked
         # with a SpellingError or SpellingSuggestion error
-        assert isinstance(self._err, SpellingSuggestion) or isinstance(
-            self._err, SpellingError
+        assert (
```

Comment:
More concise and efficient: `assert(isinstance(self._err, (SpellingSuggestion, SpellingError, RitmyndirError)))`

Reply:
Done